pkg/kapis/monitoring/v1alpha3/dashboard_template_test.sh

```diff
@@ -1,3 +1,4 @@
 #! /bin/bash
 
-curl -d '{"grafanaDashboardUrl":"https://grafana.com/api/dashboards/7362/revisions/5/download", "description":"this is a test dashboard."}' -H "Content-Type: application/json" localhost:9090/kapis/monitoring.kubesphere.io/v1alpha3/clusterdashboards/test1/template
\ No newline at end of file
+curl -d '{"grafanaDashboardUrl":"https://grafana.com/api/dashboards/7362/revisions/5/download", "description":"this is a test clusterdashboard."}' -H "Content-Type: application/json" localhost:9090/kapis/monitoring.kubesphere.io/v1alpha3/clusterdashboards/test1/template
+curl -d '{"grafanaDashboardUrl":"https://grafana.com/api/dashboards/7362/revisions/5/download", "description":"this is a test dashboard."}' -H "Content-Type: application/json" localhost:9090/kapis/monitoring.kubesphere.io/v1alpha3/namespaces/default/dashboards/test2/template
```

Comment:
Why not use a test framework?

Reply:
All right, need enhance to implement.

Reply:
> Why not use a test framework?

@wansir It is hard to add this test framework. Because it is a shell script for debug.  If we want to do that,  we can add tests in repo: kubesphere/monitoring-dashboard.   This handler imports that repo.